src/lib/holocene/calendar.svelte

```diff
@@ -38,20 +38,24 @@
 
   <div class="row">
     {#each cells as { allowed, value }, index (index)}
-      <button
-        type="button"
-        on:click={allowed && value ? () => onChange(value) : noop}
-        class="cell"
-        class:highlight={allowed && value}
-        class:disabled={!allowed}
-        class:selected={new Date(
-          date.getFullYear(),
-          date.getMonth(),
-          date.getDate(),
-        ).getTime() === new Date(year, month, value).getTime()}
-      >
-        {value || ''}
-      </button>
+      {#if value}
```

Comment:
Why are we only showing the button if it has a value? So it doesn't have an empty label? If so - maybe we should just change the empty state label down on line 54.

Reply:
Well, it's not _really_ a button at that point either. So, we really just need something to take up space.